packages/node_modules/@webex/internal-plugin-flag/src/flag.js

```diff
@@ -85,30 +84,54 @@ const Flag = WebexPlugin.extend({
   * @returns {Promise<Object>} Resolves with the activities
   * TODO: this should be implemented as a batched request when migrating to the modular sdk
   */
-  mapToActivities(flags) {
-    const activityUrls = map(flags, 'flag-item');
-
-    const params = {
-      method: 'POST',
-      api: 'conversation',
-      resource: 'bulk_activities_fetch',
-      body: {
-        activityUrls
+  async mapToActivities(flags) {
+    const mapUrlActivities = new Map();
+
+    for (const flag of flags) {
+      const convoUrlRegex = /(.*)\/activities\//;
+      const activity = flag['flag-item'];
+      const match = convoUrlRegex.exec(activity);
+
+      if (match) {
+        const url = match[1];
+        let activities = mapUrlActivities.get(url);
+
+        if (!activities) {
+          activities = [];
+          mapUrlActivities.set(url, activities);
+        }
+        activities.push(activity);
       }
-    };
+      else {
+        this.logger.warn(
+          `The activity URL has a strange format (${activity}). Ignoring it.`
```

Comment:
What's the consequence of this being ignored?

Reply:
Somebody changed the format of the activity URL. Since we have no idea about this future activity URL, I'm going to ignore it. The user wouldn't know about a flagged activity